src/module.api/block.controller.ts

```diff
@@ -0,0 +1,57 @@
+import { Controller, Get, Param, Query } from '@nestjs/common'
+import { Block, BlockMapper } from '@src/module.model/block'
+import { TransactionMapper, Transaction } from '@src/module.model/transaction'
+import { ApiPagedResponse } from '@src/module.api/_core/api.paged.response'
+import { PaginationQuery } from '@src/module.api/_core/api.query'
+
+function isNumber (str: string): boolean {
+  return /^-?\d+$/.test(str)
+}
```

Comment:
```suggestion
function isHeight (str: string): boolean {
  return /^[0-9]+$/.test(str)
}
```

no negative number and you should call it is height

Reply:
Made changes.